maat-court-data-api/src/main/java/gov/uk/courtdata/common/annotations/MaatApiReadErrorResponses.java

```diff
@@ -0,0 +1,20 @@
+package gov.uk.courtdata.common.annotations;
+
+import gov.uk.courtdata.dto.ErrorDTO;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import org.springframework.http.MediaType;
+
+import java.lang.annotation.*;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.METHOD})
+@ApiResponse(
+        responseCode = "404",
```

Comment:
The existing potential responses above show that HTTP 400 and 500 could be returned in documentation however this is just 404?